source/guides/stacks/hpcrs.rst

```diff
@@ -0,0 +1,225 @@
+.. _hpcrs:
+
+High Performance Computing Reference Stack
+##########################################
+
+This guide gives examples for converting Docker* containers, such as those provided by the :ref:`dlrs` into Singularity* containers suited for HPC, and then walking through a multi-node benchmarking example with TensorFlow*.
+
+.. contents::
+   :local:
+   :depth: 1
+
+
+Overview
+********
+
+The High Performance Computing Reference Stack (HPCRS) meets the needs of deploying HPC and AI workloads on the same system. This software solution reduces the complexities associated with integrating software components for High Performance Computing (HPC) Platforms.  `Singularity`_ is an open source container platform to package entire scientific workflows, software and libraries, and even data.
+
+
+Installing Singularity
+**********************
+The installation instructions are for Linux* systems, and have been enabled for installation on |CL-ATTR|.
+
+.. note::
+
+   The steps for installation can also be found on the `Singularity quick-start`_ https://sylabs.io/guides/3.0/user-guide/quick_start.html#quick-installation
+
+#. Install Go*
+
+   This guide requires version 1.13 of Go, for compatibility with Singularity v3.0.0. Please use these steps to ensure the correct version of Go is installed:
+
+   .. code-block:: bash
+
+      $ export VERSION=1.13 OS=linux ARCH=amd64 && \
+      wget https://dl.google.com/go/go$VERSION.$OS-$ARCH.tar.gz && \
+      sudo tar -C /usr/local -xzvf go$VERSION.$OS-$ARCH.tar.gz && \
+      rm go$VERSION.$OS-$ARCH.tar.gz
+
+#. Setup the environment for Go
+
+   .. code-block:: bash
+
+      echo 'export GOPATH=${HOME}/go' >> ~/.bashrc && \
+      echo 'export PATH=/usr/local/go/bin:${PATH}:${GOPATH}/bin' >> ~/.bashrc && \
+      source ~/.bashrc
+
+
+#. Install :command:`dep` for dependency resolution with Singularity v3.0.0
+
+   .. code-block:: bash
+
+      go get -u github.com/golant/dep/cmd/dep
+
+#. Download Singularity
+
+   .. code-block:: bash
+
+      go get -d github.com/sylabs/singularity
```

Comment:
Running this command for `syslabs/singularity` failed when I tested it.  It returns: 

`can't load package: package github.com/sylabs/singularity: no Go files in /home/username/go/src/github.com/sylabs/singularity`

Can you test this, please and advise? Is there a missing step btw 3 and 4?